rust.spec

```diff
@@ -65,18 +65,15 @@ License:        (ASL 2.0 or MIT) and (BSD and MIT)
 URL:            https://www.rust-lang.org
 
 %global rustc_package rustc-%{rust_version}-src
-Source0:        rustc-%{rust_version}-src.tar.gz
-Source100:      rust-%{rust_version}-i686-unknown-linux-gnu.tar.gz
+Source0:        rustc-%{rust_version}-src.tar.xz
+Source100:      rust-%{rust_version}-i686-unknown-linux-gnu.tar.xz
 Source200:      README.md
 
 Patch1: 0001-Set-proper-llvm-targets.patch
-Patch2: 0002-Disable-statx-for-all-builds.-JB-50106.patch
 Patch3: 0003-Scratchbox2-needs-to-be-able-to-tell-rustc-the-defau.patch
 Patch4: 0004-Force-the-target-when-building-for-CompileKind-Host.patch
 Patch5: 0005-Provide-ENV-controls-to-bypass-some-sb2-calls-betwee.patch
 Patch6: 0006-Scratchbox2-needs-to-be-able-to-tell-cargo-the-defau.patch
-Patch7: 0007-Disable-aarch64-outline-atomics-for-now.patch
-Patch8: 0008-Revert-Use-statx-s-64-bit-times-on-32-bit-linux-gnu.patch
 Patch9: 0009-Relocate-unset-tmp.patch
```

Comment:
Whilst this is good as well you could also "git mv" patch files so they would be order by number. For example Patch2 would be 0002-Scratchbox2-needs-to-be-able-to-tell-rustc-the-defau.patch, Patch3 would be 0003-Force-the-target-when-building-for-CompileKind-Host.patch, and so on.